modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -174,6 +174,13 @@ image::dashboard-built-in-infrastructure.png[Example: Infrastructure Dashboard]
 The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
 |===
 
+=== How Endpoint Use Metrics Are Gathered
+
+The built-in dashboards for apps contain charts that show endpoint metrics. The charts sort the available endpoints in a menu. When you designate a time period for the data that appears in the charts, the results shown in the endpoint charts can include multiple grouping intervals because the data is processed in intervals of time.
+As a result, the menu in a given chart usually shows top occurring endpoints and, depending on the number of endpoints, a group called *Other* that shows an aggregation of endpoints that do not occur as often for that time period. The charts show approximate counts for the most commonly occurring endpoints when the *Other* grouping appears.
+
+For example, when an endpoint occurs more often than other endpoints in one interval but occurs less often in another interval (thus sorting it into the *Other* group for that interval), the resulting chart displays lower than actual values for that endpoint, since the metrics that do not appear are grouped into the *Other* group.
```

Comment:
```suggestion
For example, when an endpoint occurs more often than other endpoints in one interval but occurs less often in another interval (thus sorting it into the *Other* group for that interval), the resulting table displays lower than actual values for that endpoint, since the metrics that do not appear are grouped into the *Other* group.
```